lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -371,13 +372,15 @@ def present_strategy(compute_api, wished_server):
         target_server = query_results[0]
 
     if server_attributes_should_be_changed(compute_api=compute_api, target_server=target_server,
-                                           wished_server=wished_server):
+                                           wished_server=wished_server) or ip_attributes_should_be_changed(compute_api=compute_api, target_server=target_server,
```

Comment:
Could you create a function? The line is very long to read. Something like : `server_or_ip_should_be_changed`

Reply:
I wrapped the two methods in a unique method associated with comments.
As the function return early, it sounds to me like a good trade off than having three different functions.